stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/channel/list/adapter/viewholder/ChannelViewHolder.kt

```diff
@@ -171,7 +172,12 @@ internal class ChannelViewHolder @JvmOverloads constructor(
     }
 
     private fun StreamUiChannelListItemForegroundViewBinding.configureAvatarView() {
-        avatarView.setChannelData(channel)
+        val otherUsers = channel.getUsers()
+        if (otherUsers.size == 1) {
```

Comment:
What about group channels with only one member? Should we treat them as direct channels or group channels?

Reply:
Good catch, do we have a method to distinguish them? The only difference is non-empty name?

Reply:
So, the check should be 
```
if (otherUsers.size == 1 && channel.name.isEmpty) {
  // show user avatar with online status
} else {
  // show channel avatar with 1+ avatars without online indicator
}
```
?

Reply:
not really, group channel can also have empty name. We can add an extension method similar to this: https://github.com/GetStream/stream-chat-android/blob/main/stream-chat-android-ui-components-sample/src/main/kotlin/io/getstream/chat/ui/sample/feature/chat/info/group/GroupChatInfoFragment.kt#L79
If channel is distinct and it has only one member (excluding current user) we can assume it's one to one conversation

Reply:
I checked flutter implementation and they are doing the same thing. Anyway, the "user.presence.changed" event is not working now.

Reply:
Created this task https://stream-io.atlassian.net/browse/CAS-700